static/js/lib/posts_test.js

```diff
@@ -95,4 +97,76 @@ describe("Post utils", () => {
       )
     })
   })
+
+  describe("mapPostListResponse", () => {
+    it("clears existing post ids if there is no existing pagination", () => {
+      const posts = makeChannelPostList(),
+        oldPosts = makeChannelPostList()
```

Comment:
Please define `posts`, `oldPosts`, and the lists of `oldPosts` and `posts` ids in a `beforeEach` function